ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp

```diff
@@ -83,7 +83,7 @@ void AirsimROSWrapper::initialize_ros()
     double update_gps_every_n_sec;
     double update_imu_every_n_sec;
     double publish_static_tf_every_n_sec;
-    nh_private_.getParam("is_vulkan", is_vulkan_);
+    nh_.getParam("ros_bridge/is_vulkan", is_vulkan_);

```

Comment:
is this by mistake? it doesn't break anything but still

Reply:
wohops wil revert this